lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2418,14 +2450,20 @@ static SILValue wrapInStruct(SILValue v, NominalTypeDecl *decl, SILBuilder &B,
   return B.createStruct(loc, silType, v);
 }
 
+/// `decl` is an stdlib numeric type represented by a struct wrapping an LLVM
```

Comment:
This comment doesn't explain what the function does. And the function name seems to be confusing.

The code seems to be extracting the Builtin type from a stdlib numeric type, instead of "getting a stdlib numeric type from decl" as the function name describes.

Reply:
Made some changes. PTAL.